app/controllers/support_interface/settings_controller.rb

```diff
@@ -2,13 +2,29 @@ module SupportInterface
   class SettingsController < SupportInterfaceController
     def activate_feature_flag
       FeatureFlag.activate(params[:feature_name])
-      flash[:success] = "Feature ‘#{params[:feature_name].humanize}’ activated"
+
+      if HostingEnvironment.production?
```

Comment:
Yes, it's fine to send this in all envs because [the slack worker will send it to a test channel in other envs](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/workers/slack_notification_worker.rb#L26-L28).